python/src/io.cpp

```diff
@@ -49,16 +53,17 @@ void io(py::module& m)
              return self.read_mesh(data_path, use_partition_from_file,
                                    ghost_mode);
            })
-      .def("read_vector",
-           [](dolfin::io::HDF5File& self, const MPICommWrapper comm,
-              const std::string data_path, bool use_partition_from_file) {
-             auto x = self.read_vector(comm.get(), data_path,
-                                       use_partition_from_file);
-             Vec _x = x.vec();
-             PetscObjectReference((PetscObject)_x);
-             return _x;
-           },
-           py::return_value_policy::take_ownership)
+      .def(
+          "read_vector",
```

Comment:
Why do you put a space here (and everywhere below)?

Reply:
Its clang-format  (Im using clang-format8), that does this.

Reply:
Might need more recent `clang-format`.